### ISSUE TITLE
Unable to replicate the results

### ISSUE THREAD

**Issue by psurya1994** [bryandlee/Tune-A-Video#1]
I attempted to replicate the results shown in the README by using the codebase (training with the config in config/sample.yml), but was unable to do so. I've attached the sample/step_400.gif generated below for this training run.

Can you help me understand what I might be missing? 

![step_400](https://user-images.githubusercontent.com/1769260/211965457-d13b7bb9-9320-494c-bd54-25166af28a45.gif)

**Comment by psurya1994** [comment]
In personal communication with the repo owner, I was told it was probably an issue with how I'm loading weights (I did make a couple of changes to my weights part of the code). Will take a closer look to ensure the weights are being loaded correctly. Closing the issue for now. Will reopen in case I still face issues.

**Comment by psurya1994** [comment]
My issue has been resolved. Thanks to @bryandlee !

The issue in my case was that I was using directly using the hugging face path for my model, instead of saving it locally (like the code does). Specifically:
`pretrained_model_path: runwayml/stable-diffusion-v1-5`. Because of which, the model was not loading the weights for the UNet correctly.

I fixed the issue by appropriately changing the path [on this line](https://github.com/bryandlee/Tune-A-Video/blob/a7679e16beeffeec388f532f04ee5dbfd0580756/train.py#L196) of the code.
